lib/anycablebility/cli/app.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'anycablebility/rpc'
+require 'anycablebility/tests'
+require 'anycablebility/client_factory'
+
+module Anycablebility
+  module Cli
+    # This class is the entry point of Anycablebility
+    class App
+      def run(
+        target: 'ws://0.0.0.0:8080/cable',
+        redis: 'redis://localhost:6379',
+        debug: false
+      )
+        logger = Logger.new(STDOUT)
+        logger.level = debug ? Logger::DEBUG : Logger::WARN
+
+        rpc = Rpc.instance.configure(redis, debug, logger)
```

Comment:
>     class << self
>       attr_writer :logger
>     end

Why do we need this? This is an attribute writer on `Logging` itself. We need an `attr_accessor :logger` on the `Anycablebility` instead. Or am I missing something?